code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -81,6 +81,7 @@
 	melee_damage_lower = 25
 	melee_damage_upper = 25
 	a_intent = INTENT_HARM
+	gold_core_spawnable = HOSTILE_SPAWN
```

Comment:
Adding this here makes a child class mob inherit it. In this case, `/mob/living/simple_animal/hostile/venus_human_trap/ghost_playable` thus adding venus man traps twice, not once. Consider if you want the click-to-play version or the normal one, and either move `gold_core_spawnable = HOSTILE_SPAWN` to the click-to-play, or add `gold_core_spawnable = NO_SPAWN` to the other one below.